queen-of-versions/CHANGELOG.md

```diff
@@ -1,10 +1,16 @@
 # Changelog
 
+## Version 0.3.2
+
+_2021-04-29_
+
+- PrinceOfVersions to version 4.0.4
+
 ## Version 0.3.1
 
 _2021-04-02_
 
-- PrinceOfVersions to version 4.0.3
+- PrinceOfVersions to version 4.0.3 (NOTE: Skip this version & use 0.3.2)
```

Comment:
Why this?
I wouldn't event write it as it is probable someone who is integrating the library would use last available version.
If you really want to write it, I would then at least explain the reasoning